curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -181,19 +181,23 @@ CURVEFS_ERROR S3ClientAdaptorImpl::Truncate(InodeWrapper *inodeWrapper,
     const auto *inode = inodeWrapper->GetInodeLocked();
     uint64_t fileSize = inode->length();
 
+    int64_t deltaBytes = 0;
```

Comment:
I think it's better to put this logic into FuseS3Client::Truncate or put FuseClient::Truncate, so that FuseVolumeClient is also supported.

Reply:
i try to reuse exist code. to achieve what you say, i think we need to rewrite the whole interface, put duplicate logic in fuseclient, rather than the lowest level code. maybe a enhancement.